ctru-rs/src/services/gfx.rs

```diff
@@ -96,14 +118,14 @@ pub struct RawFrameBuffer<'screen> {
     /// Pointer to graphics data to be rendered.
     pub ptr: *mut u8,
     /// The width of the framebuffer in pixels.
-    pub width: u16,
+    pub width: usize,
     /// The height of the framebuffer in pixels.
-    pub height: u16,
+    pub height: usize,
```

Comment:
IIRC these were `u16` since the maximum value is much smaller than `usize::MAX` but it's probably fine to have like this as long as there is some check for max values somewhere

Reply:
Yes, it's always smaller that u16::MAX. However, since this is a "read-only" member (there is no function that asks for these sizes, right?) there is no problem using it this way. The user will convert it back to `usize` 99% of times since that needed to work with pointers. It's just easier to have it like this.